### PR TITLE
Fix archive URL typo for ncdf4

### PR DESCRIPTION
**Pull request by zao**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/ncdf4/ncdf4-1.16-intel-2017a-R-3.4.0.eb [hunk 1]
```diff
@@ -11,7 +11,7 @@ toolchain = {'name': 'intel', 'version': '2017a'}
 
 source_urls = [
     'http://cran.r-project.org/src/contrib/',
-    'http://cran.r-project.org/src/contrib/Archive/$(name)s/',
+    'http://cran.r-project.org/src/contrib/Archive/%(name)s/',
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncdf4/ncdf4-1.16-intel-2017a-R-3.4.0.eb]: @zao Please change to `https` (also for `homepage` while you're at it), current PR tests are a bit stricter now...

Comment (zao) [reply easybuild/easyconfigs/n/ncdf4/ncdf4-1.16-intel-2017a-R-3.4.0.eb]: Changed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/12d53e00acccac70385269e337f28ed6 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @zao!

**Pull request merged**